agent/utils.py

```diff
@@ -63,7 +63,7 @@ def parse_results(json_output: dict[str, Any]) -> Iterator[Vulnerability]:
     for vulnerability in vulnerabilities:
         extra = vulnerability.get("extra", {})
         description = extra.get("message", "")
-        title = description.split(".")[0]
```

Comment:
Can you add unit test for it.

Reply:
this change doesn't break the existing unit test.